lib/mutant/color.rb

```diff
@@ -24,7 +24,7 @@ def format(text)
         text
       end
 
-    private
+    private # rubocop:disable AccessModifierIndentation
```

Comment:
Yeah I think I agree. I think the cop is useful overall though so I hope that doesn't translate to removing the cop too